sfnt/font_test.go

```diff
@@ -40,3 +41,59 @@ func TestSmokeTest(t *testing.T) {
 		file.Close()
 	}
 }
+
+// benchmarkParse tests the performance of a simple Parse.
+// Example run:
+//   go test -cpuprofile cpu.prof -bench . -run=^$ -benchtime=30s
+//   go tool pprof cpu.prof
+//
+// BenchmarkParseOtf-8          	 5000000	      2784 ns/op	    1440 B/op	      33 allocs/op
```

Comment:
@bramp neat! This is surprisingly low, two allocs per table + 1 ?

Reply:
the "-8" is part of the Go Benchmarking. It is calling "BenchmarkParseOtf(b *testing.B)" where b.N = 8. Go will vary the size of N to get more reliable results.